### PR TITLE
[#175245285] Add Azure deploy how-to

### PR DESCRIPTION
**Pull request by AleDore**
This PR adds a brand new section that will contains a collection of how-to for development purpose. It starts with Azure deploy how-to.

### CODE REVIEW

File: docs/io-handbook/development-howtos.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+### Monitor Production logs
+
+While Pipeline is running, you have to monitor production logs to check if there are errors during deploy or in a post deploy phase. In order to achieve this checks, you have to perform other several operations.
+
+#### Check activity log
+- Login to [Azure Portal](https://portal.azure.com/#home) and search for the resource you are interested to monitor (in this case ***io-p-fn3-assets***)
+- Select ***Activity Log*** and take care about swap operations (from staging to production) ![activity Log 1](../../static/img/activity_log1.png) ![activity log 2](../../static/img/activity_log2.png) ![activity log 3](../../static/img/activity_log3.png)
+
+#### Check Application Insights logs
+- Login to [Azure Portal](https://portal.azure.com/#home) and search for application insights (***io-p-ai-common***)
```
Comment (gunzip) [inline docs/io-handbook/development-howtos.md:53]: can you please remove payment-activations ids ? :-)

**Review by gunzip — APPROVED** [review]

File: docs/io-handbook/how-to-deploy.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+![az webapp tail](../../static/img/az-log-tail.png)
+
+## Test in production environment
+
+Once the deploy phase is finished and you are confident that there are no relevant errors in production,
+try to make some integration tests against what has been just released (ie. by calling the API directly from Postman or cURL);
+aside, check if the other core IO functionalities are still working (ie. manually triggering events using the IO mobile application).
+
+**Congratulations! You have terminated your first deploy in our production environment using Azure pipelines!**
```
Comment (balanza) [inline docs/io-handbook/how-to-deploy.md:88]: <3

File: docs/io-handbook/how-to-deploy.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+## Create a GitHub release
+
+After checking code changes you **_must_** create a new release on GitHub
+(check out if it is a minor or a major) before proceed with the next step.
+
+## Run the pipeline
+
+1. Navigate to the [Azure DevOps](https://dev.azure.com/pagopa-io/) portal
+1. Click on **_io-functions-assets_**' s project
```
Comment (balanza) [inline docs/io-handbook/how-to-deploy.md:43]: Do we want to use a placeholder for the project name?

File: docs/io-handbook/how-to-deploy.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
+---
+title: How to deploy
```
Comment (balanza) [inline docs/io-handbook/how-to-deploy.md:2]: The guide actually defines deploy procedure for runnable applications. Do we want to differentiate for different types of software? We have
* npm packages
* the App itself

Comment (AleDore) [reply docs/io-handbook/how-to-deploy.md:2]: I'd rather write other specific guides for npm packages and the App 👍

Comment (balanza) [reply docs/io-handbook/how-to-deploy.md:2]: I agree.

**Pull request merged**